core/parachain/approval/approval_distribution.hpp

```diff
@@ -729,6 +729,9 @@ namespace kagome::parachain {
     };
     SafeObject<std::unordered_map<CandidateHash, ApprovalCache>, std::mutex>
         approvals_cache_;
+
+    metrics::RegistryPtr metrics_registry_ = metrics::createRegistry();
```

Comment:
We are having a ton of fields being initialized in constructor and only this field is initialized here. 
For the sake of consistency I prefer moving its initialization to the constructor as well.